src/query-builder/aggregate-function-builder.ts

```diff
@@ -95,6 +103,43 @@ export class AggregateFunctionBuilder<DB, TB extends keyof DB, O = unknown>
     })
   }
 
+  /**
+   * Adds an `order by` clause inside the aggregate function for specify correct result order
+   * ([see postgres docs](https://www.postgresql.org/docs/9.5/sql-expressions.html#SYNTAX-AGGREGATES))
```

Comment:
This is more than enough. "Bring Your Own SQL Knowledge".

```suggestion
   * Adds an `order by` clause inside the aggregate function.
```